automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/JupyterModel.scala

```diff
@@ -2,13 +2,40 @@ package org.broadinstitute.dsde.workbench.leonardo
 
 case class ContentItem (
                        `type`: String,
-                       mimetype: String,
+                       mimetype: Option[String],
                        writable: Boolean,
                        name: String,
                        format: String,
                        created: String,
                        content: Option[String],
                        last_modified: String,
-                       path: String,
+                       path: String ,
                        size: Int
 )
+
+//TODO: the below objects are a representation of the notebook
+//The attempt to serialize all data fail failed, as the /api/contents endpoint which is used to retrieve a notebook from the jupyter-server image on the cluster changes a lot of the contents
+//If the attempt is to succeed, we should directly retrieve the file from the disk of the image on the cluster via gcloud utility functions.
+//This was not done originally, as it was deemed a size comparison was sufficient for notebook equality
+
+//response from jupyter API for an .ipynb file
+case class NotebookContentItem (
+                                 `type`: String,
```

Comment:
`size` is the only field being used currently. Talked to Justin offline and he'll either remove them in this PR or a followup PR

Reply:
Agreed about only defining the fields we're using. The json mapper should ignore unknown fields.

Reply:
will remove with subsequent automation tests